db/migrations/00001_initialise.sql

```diff
@@ -0,0 +1,108 @@
+-- +goose Up
+CREATE TABLE livestreams
+(
+    livestream_id   bigint GENERATED ALWAYS AS IDENTITY,
+    status          text        NOT NULL,
+    stream_key      text        NOT NULL,
+    title           text        NOT NULL,
+    description     text        NOT NULL,
+    scheduled_start timestamptz NOT NULL,
+    scheduled_end   timestamptz NOT NULL,
+    visibility      text        NOT NULL,
+    PRIMARY KEY (livestream_id),
+    UNIQUE (stream_key)
+);
+
+CREATE TABLE links
+(
+    link_id          bigint GENERATED ALWAYS AS IDENTITY,
+    livestream_id    integer NOT NULL,
+    integration_type text    NOT NULL,
+    integration_id   text    NOT NULL,
+    PRIMARY KEY (link_id),
+    CONSTRAINT fk_livestream FOREIGN KEY (livestream_id) REFERENCES livestreams (livestream_id),
+    UNIQUE (integration_type, integration_id)
+);
+
+CREATE TABLE rtmp_outputs
+(
+    rtmp_output_id bigint GENERATED ALWAYS AS IDENTITY,
+    output_url     text NOT NULL,
+    PRIMARY KEY (rtmp_output_id)
+);
```

Comment:
Oh wow I didn't realise these were not in their own schema, must be a left over from when I was using `sqlite`. I'm kinda tempted to move these to their own schema, but outside of scope of this PR!